client/src/pages/TakeSurvey/TakeSurvey.test.jsx

```diff
@@ -12,7 +12,8 @@ const dummySurvey = {
   description:
     'A survey description that is about 1 sentence long yeah blah blah',
   status: 'created',
-  disclaimer: 'some disclaimer',
+  disclaimer:
+    'In addition to your opinion, we will also collect your name and email address for a follow up if required. We respect your trust and protect your privacy by never sharing this data with third parties as well as ensuring this data will be stored until the research period is over. By filling out this survey, you agree that we we will process your data in line with the Beamery Privacy Policy. If you would like to chat about this policy, or if you subsequently change your mind and want us to delete your personal data, please email steven.bianchi@beamery.com.',
```

Comment:
There's a double "we" in the middle

Reply:
Could we change "as well as ensuring this data will be stored until the research period is over." to "as well as ensuring this data will not be stored after the research period has finished."

Reply:
Is it possible to link the Privacy policy to the actual policy? Then the recipients need only speak to Steve if they don't want their data stored?

Reply:
this is what was sent to us by Milo and Andrew, caught the double 'we' !

Reply:
gonna add the link